### PR TITLE
kafka offsets & update system dashboard

### PR DESCRIPTION
**Pull request by snopoke**
@calellowitz 

cc @czue 

image

### CODE REVIEW

File: corehq/apps/change_feed/topics.py [hunk 1]
```diff
@@ -35,3 +38,23 @@
 
 def get_topic(document_type_object):
     return document_type_object.primary_type
+
+
+
+def get_topic_offset(topic):
```
Comment (dimagimon) [inline corehq/apps/change_feed/topics.py]: E303 too many blank lines (3)

**Comment by czue** [comment]
build failing. looks real.
```
======================================================================
FAIL: test_iteration (corehq.util.tests.test_doc_processor.CaseResumableSqlModelIteratorTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/util/tests/test_doc_processor.py", line 152, in test_iteration
    self.assertEqual([doc["_id"] for doc in self.itr], self.all_doc_ids)
AssertionError: Lists differ: [u'ce0bfd000abd4c008e2db43cf28... != [u'fbb0737df43446de92e50dfce87...
First differing element 0:
ce0bfd000abd4c008e2db43cf28ac707
fbb0737df43446de92e50dfce8754c39
Second list contains 4 additional elements.
First extra element 5:
9c6ff74fa8694caea7fceb7bbb45eee9
+ [u'fbb0737df43446de92e50dfce8754c39',
+  u'52b7a578ae8140b491850b7a743d710b',
+  u'8449a07c35324ad6b5902f840304a2c7',
+  u'32d3e78579e143ed8bc097b8b7c6a89c',
- [u'ce0bfd000abd4c008e2db43cf28ac707',
? ^
+  u'ce0bfd000abd4c008e2db43cf28ac707',
? ^
   u'9c6ff74fa8694caea7fceb7bbb45eee9',
   u'7b42da4601d748a782f63833a887044d',
   u'015fae0dae4c46d9a55c46a079b83fca',
   u'45e40404f5ca4a7b8aa580f023ce9515']
-------------------- >> begin captured logging << --------------------
```
```======================================================================
ERROR: test_bad_integer_datatype (corehq.apps.userreports.tests.test_pillow.IndicatorPillowTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 87, in wrapper
    return func(*args, **kw)
  File "/mnt/commcare-hq-ro/corehq/apps/userreports/tests/test_pillow.py", line 98, in setUp
    self.pillow.get_change_feed().get_current_offsets()
  File "/mnt/commcare-hq-ro/corehq/apps/change_feed/consumer/feed.py", line 104, in get_current_offsets
    return get_multi_topic_offset(self.topics)
  File "/mnt/commcare-hq-ro/corehq/apps/change_feed/topics.py", line 52, in get_multi_topic_offset
    assert set(topics) < set(ALL)
AssertionError
----------------------------------------------------------------------```

File: corehq/apps/hqadmin/templates/hqadmin/system_info.html [hunk 2]
```diff
@@ -166,7 +166,17 @@
 
             self.width = function() {
                 return (self.seq * 100) / self.db_offset;
-            }
+            };
+
+            self.status = function() {
```
Comment (czue) [inline corehq/apps/hqadmin/templates/hqadmin/system_info.html]: nice

File: corehq/ex-submodules/pillowtop/utils.py [hunk 3]
```diff
@@ -136,15 +136,13 @@ def get_pillow_json(pillow_config):
     else:
         time_since_last = ''
         hours_since_last = None
-    try:
-        db_seq = pillow.get_change_feed().get_latest_change_id()
```
Comment (czue) [inline corehq/ex-submodules/pillowtop/utils.py]: I think you can remove `get_latest_change_id` now too. Pretty sure this was the only real usage.

**Comment by czue** [comment]
looks good on :white_check_mark:

**Comment by snopoke** [comment]
:white_check_mark:

**Pull request merged**